_pages/contributing.md

```diff
@@ -275,136 +303,47 @@ Jira Issue: https://gem5.atlassian.net/browse/GEM5-186
-components you have modified should be added as reviewers. These should
-correspond to the tags you included in the commit header. **Please consult
-[MAINTAINERS.yaml](
-https://github.com/gem5/gem5/blob/stable/MAINTAINERS.yaml) to
-see who maintains which component**. As an example, for a commit with a header
-of `tests,arch : This is testing the arch component` then the maintainers for
-both `tests` and `arch` should be included as reviewers.
+Now, via the GitHub web interface, you can [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) of your changes from your forked repository's branch into the gem5 `develop` branch.
 
-Reviewers will then review this change. There are three scores which the commit
-shall be evaluated: "Code-Review", "Maintainer", and "Verified".
-
-Each reviewer can give a score from `-2` to `+2` to the "Code-Review" score,
-where `+2` indicates the reviewer is 100% okay with the patch in its current
-state and `-2` when the reviewer is certain they do not want the patch
-merged in its current state.
-
-Maintainers can add `+1` or `-1` to the "Maintainer" score. A `+1` score
-indicates that the maintainer is okay with the patch.
-
-When a Maintainer gives a `+1` our continuous integration system will process
-the change. At the time of writing, the continuous integration system will run:
-
-```
-scons build/NULL/unittests.opt
-cd tests
-python main.py run
-```
+## Passing the checks
 
-If this executes successfully (i.e. the project builds and the tests pass) the
-continuous integration system will give a `+1` to the "Verifier" score, and a
-`-1` if it did not execute successfully.
+Once you have created a pull request, the gem5 Continuous Integration (CI) tests will run.
+These run a series of checks to ensure your changes are valid.
+These must pass before your changes can be merged into the gem5 `develop` branch.
 
-Gerrit will permit a commit to be merged if at least one reviewer has given a
-`+2` to the "Reviewer" score, one maintainer has given a `+1` to the
-"Maintainer" score, and the continuous integration system has given a `+1` to
-the "Verifier" score.
+In addition to the CI tests, your changes will be reviewed by the gem5 community.
+Your pull-request must have the approval of at least one community member prior to being merged.
 
-For non-trivial changes, it is not unusual for a change to receive feedback
-from reviewers that they will want incorporated before giving the commit a
-score necessary for it to be merged. This leads to an iterative process.
+Once your pull-request has passed all the CI tests and has been approved by at least one community member, it will be merged a gem5 Project Maintainer will do a [Squash and Merge](https://docs.gitlab.com/ee/user/project/merge_requests/squash_and_merge.html) on the pull-request.
```

Comment:
Wrong link :)

```suggestion
Once your pull-request has passed all the CI tests and has been approved by at least one community member, it will be merged a gem5 Project Maintainer will do a [Squash and Merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/about-pull-request-merges#squash-and-merge-your-commits) on the pull-request.
```